docs/en/pid-eaa-issuance.rst

```diff
@@ -137,9 +176,10 @@ The JWS payload of the request object is represented below:
 
 .. note::
 
-   **User Authentication and Consent:** The PID Provider performs the User authentication based on the requirements of eIDAS LoA High and asks the User consent for the PID issuance. 
+   **User Authentication and Consent:** The PID Provider performs the User authentication based on the requirements of eIDAS LoA High by means of national notified eIDAS scheme and asks the User consent for the PID issuance. 
+   The (Q)EAA Issuer performs the User authentication requesting a valid PID to the Wallet Instance. The (Q)EAA Issuer MUST use [`OpenID4VP`_] to dynamically request presentation of the PID. From a protocol perspective, the (Q)EAA Issuer then acts as a verifier and sends a presentation request to the Wallet Instance. The Wallet Instance MUST have a valid PID obtained prior to starting a transaction with the (Q)EAA Issuer.
```

Comment:
this suggests that a RP could authenticate the user in a traditional way and then offer an auth code to be used to a pre-authorized code flow

:-)

Reply:
@peppelinux can you elaborate on your comment?